installation/scripts/minikube.sh

```diff
@@ -7,6 +7,7 @@ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 MINIKUBE_DOMAIN=""
 MINIKUBE_VERSION=0.28.2
 KUBERNETES_VERSION=1.10.0
+KUBECTL_CLI_VERSION=1.10
```

Comment:
Since you don't compare major versions, you can just go with KUBECTL_CLI_VERSION_MINOR=10 and skip the line 108 entirely.